e2e-tests/commands/shared-commands/assertions/assert-contact-details-content.js

```diff
@@ -0,0 +1,20 @@
+import { emailLink, emailPrefix } from '../../../pages/shared';
+import contactDetails from '../../../partials/contactDetails';
+import { CONTACT_DETAILS } from '../../../content-strings';
+
+const { PHONE, EMAIL, OPENING_TIMES } = CONTACT_DETAILS;
+
+/**
+ * assertContactDetailsContent
+ * Check contact details content
+ */
+const assertContactDetailsContent = () => {
+  cy.checkText(emailPrefix(), `${EMAIL.PREFIX}:`);
```

Comment:
should there be something after the `:`?

Reply:
The proceeding text is checked below in `checkLink`